cli/app.go

```diff
@@ -2397,9 +2361,24 @@ Copy multiple files from the machine to a local destination with recursion and k
 									Name:     tunnelFlagDestinationPort,
 									Required: true,
 								},
-							},
+							}...),
 							Action: createCommandWithT[robotsPartTunnelArgs](RobotsPartTunnelAction),
 						},
+						{
+							Name: "motion",
+							Subcommands: []*cli.Command{
+								{
+									Name: "get-pose",
+									Flags: append(commonPartFlags, []cli.Flag{
+										&cli.StringFlag{
+											Name:     "component",
+											Required: true,
+										},
+									}...),
```

Comment:
We already have a `component-name` flag, is that not sufficient? I think generalizing the use of the existing flag (adding aliasing if necessary) is good for preventing bloat and ensuring consistency.

Reply:
Can you elaborate? I don't need this to be a blocker and I'm happy to approve at this point but to my knowledge this will be the only instance of a flag with a name hard-coded like this as opposed to using a variable (or certainly the only _intentional_ instance). I'd love to understand why for better maintaining the CLI going forward.

Reply:
Variables are good for things that change, where the string itself isn't important.
here, the string is literally the api.